boards/arm/nucleo_wb55rg/pinmux.c

```diff
@@ -44,6 +44,10 @@ static const struct pin_config pinconf[] = {
 #if DT_HAS_NODE(DT_NODELABEL(adc1))
 	{STM32_PIN_PC2, STM32WBX_PINMUX_FUNC_PC2_ADC1_IN3},
 #endif
+#ifdef CONFIG_USB_DC_STM32
```

Comment:
`#if DT_HAS_NODE(DT_NODELABEL(usb))` should be used

Reply:
Done. Thanks!